lessons/codap-bar-charts/langs/en-us/index.adoc

```diff
@@ -125,25 +131,36 @@ Explain that CODAP is designed to be student-friendly and that the interface enc
 In this section, students will develop a methodical approach to creating displays. First, demonstrate how to create a bar chart showing the `sex` breakdown of the animals. To do this, model asking yourself three important questions (below) in order to build a bar chart in CODAP.
 
 [.lesson-instruction]
-* We're going to complete @printable-exercise{pages/plot-practice.adoc} together. To make a bar chart showing the sex of animals from the shelter, I will ask myself a series of important questions.
-* Which attributes on which axes?
-** _Sex belongs on the x-axis._
-* What type of data?
-** _Male, female, and hermaphrodite are all categories. The bar chart will display categorical data._
-* What configuration?
-** _CODAP initially creates a dot plot of the data. I will need to fuse the dots into bars._
+--
+We're going to complete @printable-exercise{pages/plot-practice.adoc} together. To make a dot plot showing the sex of animals from the shelter, we ask a series of important questions:
+
+* Which **Column / Attributes** on which axes?
+** _Sex belongs on the either axis._
+* What **Type of Data**?
+** _Male, female, and hermaphrodite are all categories. The chart will display categorical data._
+* What **Configuration**?
+** _CODAP initially creates a dot plot of the data, so no special configuration is needed._
+--
 
-Focus on supporting students in learning how to pose productive questions when looking at data. Invite students to repeat the process you just modeled as they create a bar chart showing the species of animals from the shelter.
+Focus on supporting students in learning how to pose productive questions when looking at data. Invite students to repeat the process you just modeled as they create a bar chart (Q2 on the same page) that shows the species of animals from the shelter.
```

Comment:
It feels to me like the lesson plan probably should have guidance around this second question here... perhaps the following?

> * Which **Column / Attributes** on which axes?
> ** _Species belongs on the either axis._
> * What **Type of Data**?
> ** _cat, dog, etc. are all categories. The chart will display categorical data._
> * What **Configuration**?
> ** _We need to fuse the dots into bars._

Reply:
I like this, thanks!